doc/static-badges.md

```diff
@@ -0,0 +1,12 @@
+# Static Badges
```

Comment:
I'd love there to be a really good place on shields.io we could link to here. I don't think we can completely blame users for not being able to self-serve these requests when we don't really explain the feature super well. Re-vamping the site is an whole other issue (which I have also been noodling with) but adding this seems like a 'quick fix' to give us something to link to from the issue form.